data/scripts/cable_club.inc

```diff
@@ -1,20 +1,20 @@
-BattleFrontier_PokemonCenter_2F_MapScript1_276ACF: @ 8276ACF
-DewfordTown_PokemonCenter_2F_MapScript1_276ACF: @ 8276ACF
-EverGrandeCity_PokemonCenter_2F_MapScript1_276ACF: @ 8276ACF
-EverGrandeCity_PokemonLeague_2F_MapScript1_276ACF: @ 8276ACF
-FallarborTown_PokemonCenter_2F_MapScript1_276ACF: @ 8276ACF
-FortreeCity_PokemonCenter_2F_MapScript1_276ACF: @ 8276ACF
-LavaridgeTown_PokemonCenter_2F_MapScript1_276ACF: @ 8276ACF
-LilycoveCity_PokemonCenter_2F_MapScript1_276ACF: @ 8276ACF
-MauvilleCity_PokemonCenter_2F_MapScript1_276ACF: @ 8276ACF
-MossdeepCity_PokemonCenter_2F_MapScript1_276ACF: @ 8276ACF
-OldaleTown_PokemonCenter_2F_MapScript1_276ACF: @ 8276ACF
-PacifidlogTown_PokemonCenter_2F_MapScript1_276ACF: @ 8276ACF
-PetalburgCity_PokemonCenter_2F_MapScript1_276ACF: @ 8276ACF
-RustboroCity_PokemonCenter_2F_MapScript1_276ACF: @ 8276ACF
-SlateportCity_PokemonCenter_2F_MapScript1_276ACF: @ 8276ACF
-SootopolisCity_PokemonCenter_2F_MapScript1_276ACF: @ 8276ACF
-VerdanturfTown_PokemonCenter_2F_MapScript1_276ACF: @ 8276ACF
+BattleFrontier_PokemonCenter_2F_MapScript1_OnTransition: @ 8276ACF
```

Comment:
See if you can condense these into a single label, and give `OldaleTown_PokemonCenter_2F_EventScript_276AD5` a proper name.